src/main/resources/db/migration/V1__authentication.sql

```diff
@@ -1,17 +1,18 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
What is a UUID-OSSP?

Reply:
UUID-OSSP is a postgresql extension that we can enable that let's us generate UUID's directly in postgresl. So we can have

```sql
   customer_id UUID NOT NULL DEFAULT UUID()
```